lib/gourami/validations.rb

```diff
@@ -57,6 +62,14 @@ def clear_and_set_errors(new_errors)
       errors
     end
 
+    def raise_validate_errors
+      raise ValidationError.new(errors)
+    end
+
+    def handle_validation_error(error)
+      clear_and_set_errors(error.errors) unless error.errors.nil?
```

Comment:
I don't think this `unless error.errors.nil?` stuff is right.

Reply:
Are you saying that we don't need it?

Reply:
I read it wrong, I was thinking it said `self.errors.nil?`